MIGRATION.md

```diff
@@ -144,6 +145,22 @@ In v5, the workaround was removed, meaning that all components that require acce
 
 Before migrating, make sure that you add appropriate and localized labels for all occurrences flagged by the error mechanism. After that, stop setting the `UNSAFE_DISABLE_ACCESSIBILITY_ERRORS` environment variable.
 
+### noMargin prop dev-time error
+
+Since it require large scale changes in the apps, the noMargin prop is not removed in the v5, instead the components throw runtime errors in development at missing noMargin prop. Production and testing builds are not affected.
+
+During the migration you can use an escape hatch to continue running the app in development without throwing the noMargin prop errors.
+
+In your app, expose the `UNSAFE_DISABLE_NO_MARGIN_ERRORS` environment variable. You can use the [Webpack `DefinePlugin`](https://webpack.js.org/plugins/define-plugin/) ([here's an example](https://github.com/sumup-oss/circuit-ui/blob/main/.storybook/main.js#L45-L53) in the Circuit UI Storybook config) or, if your app uses Next.js, you can declare the variable in your `next.config.js` ([Next.js documentation](https://nextjs.org/docs/api-reference/next.config.js/environment-variables)).
```

Comment:
The code lines ref is broken here and it subject to change. Instead of pointing to a permalink on the `next` branch I suggest simply linking to the config file—the DefinePlugin isn't hard to find

```suggestion
In your app, expose the `UNSAFE_DISABLE_NO_MARGIN_ERRORS` environment variable. You can use the [Webpack `DefinePlugin`](https://webpack.js.org/plugins/define-plugin/) (see the [Circuit UI Storybook Webpack config](https://github.com/sumup-oss/circuit-ui/blob/main/.storybook/main.js) as an example) or, if your app uses Next.js, you can declare the variable in your `next.config.js` ([Next.js documentation](https://nextjs.org/docs/api-reference/next.config.js/environment-variables)).
```